imagevector/images.yaml

```diff
@@ -6,5 +6,5 @@
 images:
 - name: oidc-webhook-authenticator
   sourceRepository: github.com/gardener/oidc-webhook-authenticator
-  repository: eu.gcr.io/gardener-project/gardener/oidc-webhook-authenticator
+  repository: europe-docker.pkg.dev/gardener-project/releases/gardener/oidc-webhook-authenticator
```

Comment:
/hold

This change requires https://github.com/gardener/oidc-webhook-authenticator/pull/143 to be merged, released and delivered firstly.

Reply:
@vpnachev : I _did_ cp referenced images..

Reply:
The change is fine, it just needs to wait OWA to be released and delivered into this repo.